cmd/vicadmin/logs.go

```diff
@@ -73,19 +78,22 @@ func (r dlogReader) open() (entry, error) {
 		return nil, err
 	}
 
-	// DiagnosticManager::DEFAULT_MAX_LINES_PER_BROWSE = 1000
-	start := h.LineEnd - 1000
-
-	h, err = m.BrowseLog(ctx, r.host, r.name, start, 0)
-	if err != nil {
-		return nil, err
-	}
+	end := h.LineEnd
+	start := end - logLines
 
 	var buf bytes.Buffer
+	for start < end {
+		h, err = m.BrowseLog(ctx, r.host, r.name, start, lines)
+		if err != nil {
+			return nil, err
+		}
+
+		for _, line := range h.LineText {
+			buf.WriteString(line)
+			buf.WriteByte(0x0a)
```

Comment:
I would you a different form for byte encoding. If you use single quotes it means byte. So, '\n' is enough. Otherwise it might be a little confusing for people who do not remember ASCII table.